src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -131,6 +135,24 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       `
     );
   }
+  /**
+   * Queries the microsoft graph for a user based on the user id and adds them to the selectedPeople array
+   *
+   * @param {[MicrosoftGraph.User]} an array of user ids to add to selectedPeople
+   * @returns {Promise<MicrosoftGraph.User>}
+   * @memberof MgtPeoplePicker
+   */
+  public async selectUsersById(people) {
```

Comment:
update function signature to include the argument and return types. The `people` attribute should be named `userIds`;

Reply:
updated to string value (as getUser only accepts string): 
9324535